python/src/aiconfig/ChatCompletion.py

```diff
@@ -199,3 +203,135 @@ async def run_and_await_serialize():
     else:
         serialized_prompts = asyncio.run(run_and_await_serialize())
     return serialized_prompts
+
+
+# TODO type this
+def create_and_save_to_config_v2(
```

Comment:
if only v2 is used, can we get rid of the v1? can do on top

Reply:
only place its used was in the example workbook, and the default export in root __init__.py. Its no longer used now with your v2. It should be safe to remove